File: Python/Contraseña.py
acceso = False
clave = "Hola"
intentos = 3

while(intentos > 0 and not acceso):
    clave = input("Ingrese la contraseña: ")

    if clave == "Hola":
        acceso = True
        print("Acceso concedido")
    else:
        intentos -= 1
        print("Contraseña incorrecta. Intentos restantes:", intentos)

if not acceso:
    print("Acceso denegado. Has agotado todos los intentos.")

File: aJava/src/Contraseña.java
import java.util.Scanner;

void main() {
    try (Scanner scanner = new Scanner(System.in)) {
        boolean acceso = false;

        String contraseña = "Hola";
        
        int intentos = 3;
        
        while(intentos > 0 && !acceso){
            System.out.print("Introduce la contraseña: ");
            String entrada = scanner.nextLine();
            
            if (entrada.equals(contraseña)) {
                acceso = true;
                System.out.println("Acceso concedido.");
            } else {
                intentos--;
                System.out.println("Contraseña incorrecta. Intentos restantes: " + intentos);
            }
        }

        if (!acceso) {
            System.out.println("Acceso bloqueado.");
        }
    }
}